### PR TITLE
refactor transition benchmark table

### PR DESCRIPTION
**Pull request by gmrabian**
### Description

Refactor code

Feel free to add on or suggest changes!

Note: this is stacked on a pdf table fix branch that is related to ticket CMDCT-3151

### Related ticket(s)

CMDCT-none

---
### How to test

- Log in as a state user
- Create a WP
- View the PDF
- Update the Transition benchmarks and refresh the pdf to see it match your changes
- Verify nothing broke

---
### Author checklist

- [x] I have performed a self-review of my code
---

### CODE REVIEW

**Review by keeysnc — APPROVED** [review]
@gmrabian thanks for cleaning this up! It seems like everything looks good!

**Review by ntsummers1 — APPROVED** [review]

**Comment by ntsummers1** [comment]
Nice job on this refactor. Beautiful work Garrett!

File: services/ui-src/src/components/tables/Table.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const Table = ({
                   key={cell + index}
                   sx={{
                     tableCell: border ? sx.tableCellBorder : sx.tableCell,
-                    color: cell == "Not Answered" ? "palette.error_darker" : "",
+                    color: cell == "Not answered" ? "palette.error_darker" : "",
```
Comment (ntsummers1) [inline services/ui-src/src/components/tables/Table.tsx:65]: Nit: Could use the constant notAnsweredText for this

Comment (gmrabian) [reply services/ui-src/src/components/tables/Table.tsx]: good idea! made this update, and in the export component as well

**Review by ailZhou — APPROVED** [review]
LGTM, Thank you for doing this refactor!

**Review by karla-vm — APPROVED** [review]

**Pull request merged**